instance/models/mixins/openedx_custom_static_pages.py

```diff
@@ -0,0 +1,49 @@
+# -*- coding: utf-8 -*-
+#
+# OpenCraft -- tools to aid developing and hosting free software projects
+# Copyright (C) 2015-2020 OpenCraft <contact@opencraft.com>
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+"""
+Instance app model mixins -  custom pages
+"""
+
+# Imports #####################################################################
+
+from django.contrib.postgres.fields import JSONField
+from django.db import models
+
+from instance.schemas.static_templates_overrides import static_templates_overrides_schema_validate
+
+# Classes #####################################################################
+
+
+class OpenEdXCustomStaticPagesMixin(models.Model):
+    """
+    Mixin to provide support for custom static pages and overridden built-in static templates.
+    """
+
+    class Meta:
+        abstract = True
+
+    static_templates_overrides = JSONField(
+        verbose_name='Draft static templates overrides JSON',
+        validators=[static_templates_overrides_schema_validate],
+        null=True,
+        blank=True,
+        default=None,
+        help_text=("The final static templates' override data committed by the user for deployment. This should be"
```

Comment:
Nit: Can you use consistent quotation marks and add a line break on this string?

Reply:
@giovannicimolin, I default to using the single quotes for strings and in this case I had to switch to using the double quotes for the string because there is a single quote within the string and I didn't want to escape it.

> add a line break on this string?

I don't see line breaks used anywhere in the help text for the fields. Even the `deploy_simple_theme` field, which has a significantly longer help text and the `theme_config` field which is of a comparable length (and from where I copied the help text format) don't have line breaks. So is this really required?

Reply:
@giovannicimolin, I have updated the PR address all the review comments except this one as you may not have had a chance to see my reply yet.

Reply:
These changes are not required at all! 
I was just going over the code and commented on everything a little bit off, but it's okay to keep the implementation here to leave the code consistent.